HOTFIX.md

```diff
@@ -0,0 +1,13 @@
+# Hotfix Process
+
+1) Build the image that you would like to use for your hotfix:
+
+`./gradlew jibDockerBuild --image=gcr.io/terra-kernel-k8s/terra-workspace-manager:hotfix-<DATE> -Djib.console=plain`
```

Comment:
Don't need to override jib.console, that's just to stop it from trying to show a progress bar in CI. You might also put these in proper code blocks, like
\`\`\`sh
./gradlew jibDockerBuild --image=gcr.io/terra-kernel-k8s/terra-workspace-manager:hotfix-<DATE>
\`\`\`
so they look like
```sh
./gradlew jibDockerBuild --image=gcr.io/terra-kernel-k8s/terra-workspace-manager:hotfix-<DATE>
```